pandas/core/reshape/melt.py

```diff
@@ -333,14 +333,11 @@ def wide_to_long(df, stubnames, i, j, sep="", suffix=r'\d+'):
     ...                    'X' : np.random.randint(3, size=3)})
     >>> df['id'] = df.index
     >>> df # doctest: +NORMALIZE_WHITESPACE, +ELLIPSIS
-       A(quarterly)-2010  A(quarterly)-2011  B(quarterly)-2010  ...
-    0           0.548814           0.544883           0.437587  ...
-    1           0.715189           0.423655           0.891773  ...
-    2           0.602763           0.645894           0.963663  ...
-       X  id
-    0  0   0
-    1  1   1
-    2  1   2
+       A(quarterly)-2010  A(quarterly)-2011  B(quarterly)-2010  ...  X  id
```

Comment:
Is the output too wide to include all the columns without any ellipses?

Reply:
Yes @TomAugspurger the line is too wide to include all columns, so deleted only "B(quarterly)-2011" column (88 > 79 chars on `pycodestyle` and `flake8`)

Reply:
Hrm, that's a little unfortunate as all the columns are important :/

Well, for now, let's use ellipses. FYI, your spacing is just slightly off, so it failed again :)


```diff
diff --git a/pandas/core/reshape/melt.py b/pandas/core/reshape/melt.py
index cec4c609c..df144db50 100644
--- a/pandas/core/reshape/melt.py
+++ b/pandas/core/reshape/melt.py
@@ -334,9 +334,9 @@ def wide_to_long(df, stubnames, i, j, sep="", suffix=r'\d+'):
     >>> df['id'] = df.index
     >>> df # doctest: +NORMALIZE_WHITESPACE, +ELLIPSIS
        A(quarterly)-2010  A(quarterly)-2011  B(quarterly)-2010  ...  X  id
-    0           0.548814           0.544883           0.437587  ...  0  0
-    1           0.715189           0.423655           0.891773  ...  1  1
-    2           0.602763           0.645894           0.963663  ...  2  2
+    0           0.548814           0.544883           0.437587  ...  0   0
+    1           0.715189           0.423655           0.891773  ...  1   1
+    2           0.602763           0.645894           0.963663  ...  1   2
 
     >>> pd.wide_to_long(df, ['A(quarterly)', 'B(quarterly)'], i='id',
     ...                 j='year', sep='-')

```

I think you need an extra space so that the values in `id` are under the `d`.